NewKernel_d/include/CGAL/NewKernel_d/Cartesian_filter_K.h

```diff
@@ -63,6 +64,12 @@ struct Cartesian_filter_K : public Base_
     typedef typename Store_kernel<EK_>::reference_type EK_rt;
     EK_rt exact_kernel()const{return sek.kernel();}
 
+
+    // Added to be similar to CGAL::Filtered_kernel
+    typedef Kernel_d_interface<EK> Exact_kernel;
```

Comment:
Note to @mglisse : think if there is a better place to put this, using Kernel_d_interface at this low level doesn't look right. Maybe in Kernel_d_interface, I can check if there is a type Exact_kernel, and in that case override it with a typedef using Kernel_d_interface (recursively)?